### DISCUSSION TITLE
For hidden apps, show icon without window title

### DISCUSSION THREAD

**Discussion by ddang94** [lawand-dot-io/taskbar#13] [Feature requests]
When a window is hidden, instead of still showing it as a greyed out task with () brackets, show it as only an icon without text. This saves some space in the taskbar

**Comment by lawand-dot-io** [comment]
At the moment there is an option in the preferences window to never show windows marked as hidden. Also, there's an option to blacklist some apps. So these are similar solutions for the moment.

**Comment by lawand-dot-io** [comment]
I split this feature request into two. This one and a new one for the button or menu that shows a list of the hidden windows: https://github.com/lawand-dot-io/taskbar/discussions/28

**Comment by lawand-dot-io** [comment]
Another similar option that's already existing is to disable the "Show windows marked as hidden" option in preferences, then choose the "Keep in Taskbar" option for an app then hide that app. The result is that when that app is hidden, its "Keep in Taskbar" button is shown and can unhide that app when clicked. The drawback to this solution is that all other hidden apps are not shown in Taskbar unless also "Keep in Taskbar" was selected for them.
